lib/pages/portal/view/website_view.dart

```diff
@@ -342,3 +328,35 @@ class _WebsiteViewState extends State<WebsiteView> {
     );
   }
 }
+
+class WebsiteIcon extends StatelessWidget {
+  const WebsiteIcon({this.website, this.canEdit, this.size, this.onTap});
+
+  final Website website;
+  final bool canEdit;
+  final double size;
+  final Function onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: StorageProvider.findImageUrl(context, website.iconPath),
+      builder: (context, snapshot) {
+        // ignore: prefer_typing_uninitialized_variables, inference_failure_on_uninitialized_variable
```

Comment:
Again, don't just ignore the warnings so they go away... You need to provide the type of the `image` variable here.